swap/src/cli.rs

```diff
@@ -107,7 +107,7 @@ pub enum Command {
         // TODO: The listen address is only relevant for Alice, but should be role independent
         //  see: https://github.com/comit-network/xmr-btc-swap/issues/77
         #[structopt(
-            short = "a",
+            short = "la",
```

Comment:
```suggestion
            short = "pa",
```
`l` is not even present in the `long` version.

Reply:
Crap, was looking at the variable name when creating the short-code 😅
Maybe we should just get rid of the short-codes altogether, they have caused nothing but trouble...